tests/issues.targets

```diff
@@ -561,8 +561,5 @@
         <ExcludeList Include="$(XunitTestBinBase)\JIT\Regression\CLR-x86-JIT\V2.0-Beta2\b409748\b409748\b409748.cmd">
              <Issue>needs triage</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)\JIT\opt\Tailcall\TailcallVerifyWithPrefix\TailcallVerifyWithPrefix.cmd">
-             <Issue>needs triage</Issue>
-        </ExcludeList>
     </ItemGroup>
```

Comment:
This test is also excluded above, in the RyuJIT/x86 section. Can you remove it from that list, also? (that is, does your fix also fix the RyuJIT/x86 case?)